releasenotes/notes/non-x86_ibm_cpu-493e51313ba222a6.yaml

```diff
@@ -0,0 +1,21 @@
+---
+features:
+  - |
+    Added partial support for running on ppc64le and s390x Linux platforms.
+    This release will start publishing pre-compiled binaries for ppc64le and
+    s390x Linux platforms on all Python versions. However, unlike other
+    supported platforms not all of Qiskit's upstream dependencies support these
+    platforms yet. So a C/C++ compiler may be required to build and install
+    these dependencies and a simple ``pip install qiskit-aer`` with just a
+    working Python environment will not be sufficient to install Qiskit Aer.
+    Additionally, these same constraints prevent us from testing the
+    pre-compiled wheels before publishing them, so the same guarantees around
+    platform support that exist for the other platforms don't apply to these
+    platforms.
```

Comment:
Shall we point to the ["tiered support" table](https://qiskit.org/documentation/getting_started.html#platform-support) for this too?  We approximately support s390x and ppc64le for Terra too, though I don't think we've got them in the table yet.